src/test/java/org/mockitousage/stubbing/CloningParameterTest.java

```diff
@@ -26,12 +27,12 @@ public void shouldVerifyEvenIfArgumentsWereMutated() throws Exception {
         businessLogic(emailSender);
 
         // then
-        verify(emailSender).sendEmail(1, new Person("Wes"));
+        verify(emailSender).sendEmail(eq(1), eq(new Person("Wes")), (Person)any(), (Person[])any());
```

Comment:
Could you revert these changes and instead add a new method with multiple arguments. (and thus add a new test to use that method separately)